Install-Guide/Community_Packages.md

```diff
@@ -5,59 +5,59 @@
 An **X** means packages are (or will be) provided in the respective repository.  
 A **—** means no plans to build new updates. Existing packages will remain in the repos.  
 
-|              |              |  4.1 ltm¹ |  4.0 stm  | 3.12 ltm  | 3.10 ltm  |
-|--------------|--------------|:---------:|:---------:|:---------:|:---------:|
-|CentOS Storage|el6           |     X³    |     X³    |     X     |     X     |
-|SIG[1]        |el7           |     X     |     X     |     X     |     X     |
-|              |              |           |           |           |           |
-|Fedora[2]     |F26           |     —     |     X     |     X     |     X²    |
-|              |F27           |     X     |     X     |     X²    |     X     |
-|              |F28           |     X     |     X²    |     —     |     —     |
-|              |F29(rawhide)  |     X     |     X²    |     —     |     —     |     
-|              |              |           |           |           |           |
-|Debian[2]     |Jessie/8      |     —     |     —     |     X     |     X     |
-|              |Stretch/9     |     X⁴    |     X⁴    |     X     |     X     |
-|              |Buster/10(Sid)|     X⁴    |     X⁴    |     X     |     X     |
-|              |              |           |           |           |           |
-|Ubuntu        |Trusty/14.04  |     —     |     —     |     —     |     X     |
-|Launchpad[3]  |Xenial/16.04  |     X⁴    |     X⁴    |     X     |     X     |
-|              |Artful/17.10  |     X⁴    |     X⁴    |     X     |     X     |
-|              |Bionic/18.04  |     X⁴    |     X⁴    |     X     |     X     |
-|              |Cosmic/18.10  |     X⁴    |     X⁴    |     —     |     —     |
-|              |              |           |           |           |           |
-|OpenSuSE Build|SLES12SP3     |     X     |     X     |     X     |     X     |
-|Service[4]    |Leap42.3      |     X     |     X     |     X     |     X     |
-|              |Leap15        |     X     |     X     |     —     |     —     |
-|              |SLES15        |     X     |     X     |     —     |     —     |
-|              |Tumbleweed    |     X     |     X     |     —     |     —     |
+|              |              |  4.1 ltm  | 3.12 ltm  | 3.10 ltm  |
+|--------------|--------------|:---------:|:---------:|:---------:|
+|CentOS Storage|el6           |     X³    |     X     |     X     |
```

Comment:
A suggestion: This is rendered on two lines/rows of the table. It might be slightly more clear if `CentOS Storage SIG[1]` was on a single table row.

Ditto for `OpenSUSE Build Service[4]` (below) and in `Related Packages` section.